src/CoreBundle/private/library/classes/TViewParser/TViewPathManager.class.php

```diff
@@ -188,15 +191,17 @@ public function getObjectPackageViewPath($sViewName, $sSubType = '', $sType = 'C
             return $sTemplatePathFromTheme;
         }
 
-        if ('Core' != $sType && false == TGlobal::IsCMSMode()) {
+        $sPath = TPkgViewRendererSnippetDirectory::PATH_OBJECTVIEWS;
```

Comment:
We shouldn't introduce Hungarian notation for new variables (please rename the variable both here and above).

Reply:
change here and above